compute_tools/src/bin/fast_import.rs

```diff
@@ -68,6 +70,8 @@ struct Spec {
     encryption_secret: EncryptionSecret,
     #[serde_as(as = "serde_with::base64::Base64")]
     source_connstring_ciphertext_base64: Vec<u8>,
+    #[serde_as(as = "Option<serde_with::base64::Base64>")]
+    restore_connstring_ciphertext_base64: Option<Vec<u8>>,
```

Comment:
nit: Why use `restore` in the name. It's leaking an implenetation detail. I'd choose `destination` instead.

Reply:
I agree here, but let's solve this with a comment? Since it's a part of the spec, that is already matched in the control plane :(

Reply:
it's planned to go on prod this week :(

Reply:
I decided as I already refactored this cli into subcommands, I can rename it as well. Renamed everywhere to `destination_*`